apps/docs/content/guides/cli/seeding-your-database.mdx

```diff
@@ -50,12 +50,42 @@ COPY countries (name, code) FROM 'seed/countries.csv' WITH (FORMAT csv);
 
 ## Generating seed data
 
-You can generate seed data for local development using [Snaplet](https://www.snaplet.dev/).
+You can generate seed data for local development using [Snaplet](https://github.com/snaplet/seed).
 
-If this is your first time using Snaplet to seed your project, you'll need to setup Snaplet
-for your project with `npx @snaplet/seed init`, and follow the interactive prompts. The command
-will look at your database and its structure, and use it to generate the configuration files
-and assets needed by `@snaplet/seed`.
+<Admonition type="tip">
+  To use Snaplet, you need to have Node.js and npm installed. You can add Node.js to your project by
+  running `npm init -y` in your project directory.
+</Admonition>
+
+If this is your first time using Snaplet to seed your project, you'll need to set up Snaplet with the following command:
+
+```bash
+npx @snaplet/seed init
+```
+
+This command will analyze your database and its structure, and then generate a JavaScript client which can be used to define exactly how your data should be generated using code. The `init` command generates a configuration file, `seed.config.ts` and an example script, `seed.ts`, as a starting point.
+
+<Admonition type="tip">
+  During `init` if you are not using an Object Relational Mapper (ORM) or your ORM is not in the
+  supported list, choose `node-postgres`.
+</Admonition>
+
+In most cases you only want to generate data for specific schemas or tables. This is defined with `select`.
+Here is an example `seed.config.ts` configuration file:
```

Comment:
```suggestion
In most cases you only want to generate data for specific schemas or tables. This is defined with `select`. Here is an example `seed.config.ts` configuration file:
```

Even though ignored single line breaks are standard-ish markdown, it doesn't work with our mdx configuration, it will show up in the output as a weird line break